vespa-hadoop/src/main/java/com/yahoo/vespa/hadoop/pig/VespaDocumentOperation.java

```diff
@@ -80,6 +81,7 @@ public static boolean valid(String text) {
     private static final String PARTIAL_UPDATE_ASSIGN = "assign";
     private static final String PARTIAL_UPDATE_ADD = "add";
     private static final String PARTIAL_UPDATE_REMOVE = "remove";
+    private static boolean printId;
```

Comment:
To be consequent with other code here, move this to the section below for other properties that are retrieved, e.g. line 104ish. Also, make this a `private final boolean`, not a `static`. I guess, to be consequent, call this `verbose` so it can be used for other things as well. So, in sum, `private final boolean verbose;`.